Quasar App Extensions using Quasar v2 ui kit,  Error: Cannot find module 'vue/compiler-sfc'

### What happened?

After Installing Quasar App Extention using Quasar v2, install modules and run project ini folder ui, get error
```
(node:33996) UnhandledPromiseRejectionWarning:   Error: Cannot find module 'vue/compiler-sfc'
  Require stack:
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\vue-loader\dist\index.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\lib\webpack\create-chain.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\lib\webpack\index.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\lib\quasar-conf-file.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\lib\cmd\dev.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\bin\quasar

  - loader.js:902 Function.Module._resolveFilename
    internal/modules/cjs/loader.js:902:15

  - loader.js:746 Function.Module._load
    internal/modules/cjs/loader.js:746:27

  - loader.js:974 Module.require
    internal/modules/cjs/loader.js:974:19

  - helpers.js:92 require
    internal/modules/cjs/helpers.js:92:18

  - index.js:8 Object.<anonymous>
    [ui]/[vue-loader]/dist/index.js:8:24

  - loader.js:1085 Module._compile
    internal/modules/cjs/loader.js:1085:14

  - loader.js:1114 Object.Module._extensions..js
    internal/modules/cjs/loader.js:1114:10

  - loader.js:950 Module.load
    internal/modules/cjs/loader.js:950:32

  - loader.js:790 Function.Module._load
    internal/modules/cjs/loader.js:790:14

  - loader.js:974 Module.require
    internal/modules/cjs/loader.js:974:19


(Use `node --trace-warnings ...` to show where the warning was created)
(node:33996) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:33996) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.`
```

### What did you expect to happen?

Not Compile Error

### Reproduction URL

-

### How to reproduce?

-

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

App Extension API

### Platforms/Browsers

_No response_

### Quasar info output

_No response_

### Relevant log output

```Shell
yarn run v1.22.5
$ cd dev && yarn dev && cd ..
$ quasar dev

 .d88888b.
d88P" "Y88b
888     888
888     888 888  888  8888b.  .d8888b   8888b.  888d888
888     888 888  888     "88b 88K          "88b 888P"
888 Y8b 888 888  888 .d888888 "Y8888b. .d888888 888
Y88b.Y8b88P Y88b 888 888  888      X88 888  888 888
 "Y888888"   "Y88888 "Y888888  88888P' "Y888888 888
       Y8b


 Dev mode.................. spa
 Pkg quasar................ v2.6.6
 Pkg @quasar/app-webpack... v3.5.1
 Pkg webpack............... v5
 Debugging................. enabled

 Configured browser support (>= 87.54% of global marketshare):
 · Chrome for Android >= 101
 · Firefox for Android >= 100
 · Android >= 101
 · Chrome >= 92
 · Edge >= 98
 · Firefox >= 91
 · iOS >= 12.2-12.5
 · Opera >= 82
 · Safari >= 13.1


 App • ⚠️  ️️Setting port to closest one available: 8081

(node:33996) UnhandledPromiseRejectionWarning:   Error: Cannot find module 'vue/compiler-sfc'
  Require stack:
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\vue-loader\dist\index.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\lib\webpack\create-chain.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\lib\webpack\index.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\lib\quasar-conf-file.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\lib\cmd\dev.js
  - C:\Users\Lawencon\Documents\tesappext\poke-card\ui\node_modules\@quasar\app-webpack\bin\quasar

  - loader.js:902 Function.Module._resolveFilename
    internal/modules/cjs/loader.js:902:15

  - loader.js:746 Function.Module._load
    internal/modules/cjs/loader.js:746:27

  - loader.js:974 Module.require
    internal/modules/cjs/loader.js:974:19

  - helpers.js:92 require
    internal/modules/cjs/helpers.js:92:18

  - index.js:8 Object.<anonymous>
    [ui]/[vue-loader]/dist/index.js:8:24

  - loader.js:1085 Module._compile
    internal/modules/cjs/loader.js:1085:14

  - loader.js:1114 Object.Module._extensions..js
    internal/modules/cjs/loader.js:1114:10

  - loader.js:950 Module.load
    internal/modules/cjs/loader.js:950:32

  - loader.js:790 Function.Module._load
    internal/modules/cjs/loader.js:790:14

  - loader.js:974 Module.require
    internal/modules/cjs/loader.js:974:19


(Use `node --trace-warnings ...` to show where the warning was created)
(node:33996) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:33996) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```


### Additional context

_No response_

Exact same error here
I started fresh ui app extension in local machine and got the same error when run yarn dev.
In fact, when I runned yarn dev first time I got this message:

```
$ yarn dev
yarn run v1.22.17
$ cd dev && yarn dev && cd ..
$ quasar dev

 App • Missing Vue deps (no longer auto supplied by @quasar/app). Installing...
 App • [sync] Running "yarn add vue@^3.0.0 vue-router@^4.0.0"

info No lockfile found.
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
[4/5] Linking dependencies...
[5/5] Building fresh packages...
success Saved lockfile.
success Saved 15 new dependencies.
info Direct dependencies
├─ vue-router@4.0.15
└─ vue@3.2.36
info All dependencies
├─ @vue/compiler-sfc@3.2.36
├─ @vue/devtools-api@6.1.4
├─ @vue/reactivity-transform@3.2.36
├─ @vue/reactivity@3.2.36
├─ @vue/runtime-core@3.2.36
├─ @vue/runtime-dom@3.2.36
├─ @vue/server-renderer@3.2.36
├─ csstype@2.6.20
├─ nanoid@3.3.4
├─ picocolors@1.0.0
├─ postcss@8.4.14
├─ source-map-js@1.0.2
├─ sourcemap-codec@1.4.8
├─ vue-router@4.0.15
└─ vue@3.2.36

 App • Installed Vue deps. Now please run your Quasar command again.

Done in 13.44s.
```
And then in the second execution of yarn dev I've gotten same @abdulrahmat97dev 's error: Error: Cannot find module 'vue/compiler-sfc'

--- a/create-quasar/templates/ui-kit/quasar-v2/BASE/ui/_package.json
+++ b/create-quasar/templates/ui-kit/quasar-v2/BASE/ui/_package.json
@@ -43,6 +43,8 @@
     "@rollup/plugin-node-resolve": "^11.2.1",
     "@rollup/plugin-replace": "^2.4.2",
     "uglify-js": "^3.13.3",
+    "vue": "^3.0.0",
+    "vue-router": "^4.0.0",
     "zlib": "^1.0.5"
   },
   "browserslist": [

--- a/create-quasar/templates/ui-kit/quasar-v2/BASE/ui/src/index.sass
+++ b/create-quasar/templates/ui-kit/quasar-v2/BASE/ui/src/index.sass
@@ -1,3 +1,7 @@
 @import 'quasar/src/css/variables.sass'
+<% if (features.component) { %>
 @import './components/Component.sass'
+<% } %>
+<% if (features.directive) { %>
 @import './directives/Directive.sass'
+<% } %>

--- a/create-quasar/utils/index.js
+++ b/create-quasar/utils/index.js
@@ -11,7 +11,7 @@ const spawn = require('child_process').spawn
 
 const logger = require('./logger')
 
-const TEMPLATING_FILE_EXTENSIONS = [ '', '.json', '.js', '.ts', '.vue', '.md', '.html' ]
+const TEMPLATING_FILE_EXTENSIONS = [ '', '.json', '.js', '.ts', '.vue', '.md', '.html', '.css', '.scss', '.sass' ]
 
 module.exports.join = join
 module.exports.logger = logger

--- a/create-quasar/utils/index.js
+++ b/create-quasar/utils/index.js
@@ -11,7 +11,7 @@ const spawn = require('child_process').spawn
 
 const logger = require('./logger')
 
-const TEMPLATING_FILE_EXTENSIONS = [ '', '.json', '.js', '.ts', '.vue', '.md', '.html', '.css', '.scss', '.sass' ]
+const TEMPLATING_FILE_EXTENSIONS = [ '', '.json', '.js', '.ts', '.vue', '.md', '.html', '.sass' ]
 
 module.exports.join = join
 module.exports.logger = logger

